pkg/util/util.go

```diff
@@ -404,24 +405,25 @@ func chooseHostInterfaceNativeGo() (net.IP, error) {
 				return nil, err
 			}
 			if len(addrs) > 0 {
-				// This interface should suffice.
-				break
+				for _, addr := range addrs {
+					if addrIP, _, err := net.ParseCIDR(addr.String()); err == nil {
+						if addrIP.To4() != nil {
+							ip = addrIP.To4()
+							break
+						}
+					}
+				}
+				if ip != nil {
```

Comment:
I think that this dead code.  I think that the only way for ip to be non-nil here would be with the above assignment stmt, in which case we would have break'd already.